velox/docs/functions/presto/conversion.rst

```diff
@@ -379,6 +380,8 @@ Valid examples
   SELECT cast('true' as boolean); -- true (case insensitive)
   SELECT cast('f' as boolean); -- false (case insensitive)
   SELECT cast('false' as boolean); -- false (case insensitive)
+  SELECT cast('F' as boolean); -- false (case insensitive)
+  SELECT cast('T' as boolean); -- true (case insensitive)
```

Comment:
nit: These two examples are redundant to the examples of 't' and 'f' since they are marked case insensitive?

Reply:
I see no harm in having this to ensure that they are indeed case insensitive.